package.json

```diff
@@ -18,13 +18,16 @@
   "scripts": {
     "bench": "sanctuary-benchmark",
     "build": "rollup -c",
-    "clean": "rimraf npm-debug.log coverage index.js index.test.js",
+    "clean": "rimraf npm-debug.log coverage index.js index.test.js .esm-cache .nyc_output node_modules/.cache",
     "lint": "eslint src test index.es.js README.md",
     "lint:readme": "remark --no-stdout --frail -u remark-validate-links README.md",
     "release": "npm outdated --long; xyz --edit --repo git@github.com:fluture-js/Fluture.git --tag 'X.Y.Z' --script scripts/distribute --increment",
-    "test": "npm run lint && npm run lint:readme && npm run test:unit && npm run test:types",
-    "test:unit": "npm run build && nyc --include src mocha --require @std/esm --ui bdd --reporter list --check-leaks --full-trace test/**.test.js",
-    "test:coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
+    "test": "npm run lint && npm run lint:readme && npm run test:unit && npm run test:types && npm run test:build",
+    "test:unit": "npm run clean && mocha --require @std/esm --ui bdd --reporter list --full-trace --check-leaks --bail test/**.test.js",
+    "test:build": "npm run clean && npm run build && mocha --require @std/esm --ui bdd --reporter dot --bail test/**.buildtest.js",
+    "test:coverage": "npm run clean && nyc --include src mocha --require @std/esm --ui bdd --reporter dot test/**.test.js || true",
+    "coverage:upload": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
+    "coverage:report": "nyc report --reporter=html",
```

Comment:
These changes facilitate potentially failing tests while testing for coverage -- annoyingly when testing `toString`, sometimes the coverage instrumentation breaks expected output.